urlparse4/cgurl.pyx

```diff
@@ -221,14 +223,14 @@ class SplitResultNamedTuple(tuple):
         return stdlib_urlunsplit(self)
 
 
-def urlsplit(url):
+def urlsplit(url, scheme='', allow_fragments=True):
```

Comment:
Is ``allow_fragments`` used?

Reply:
Do you mean is it used in Scrapy? I included it because it is used in function urlparse (https://github.com/python/cpython/blob/master/Lib/urllib/parse.py#L373). I don't really know if we should include it in this project since allow_fragments' only use is combining fragments with query.